libs/blocks/quiz/quizcontainer.js

```diff
@@ -1,11 +1,6 @@
 import { html } from '../../deps/htm-preact.js';
 
-export const DecorateBlockBackgroundCmp = ({ background = '' }) => html`<picture>
-        <source type="image/webp" srcset=${background} media="(min-width: 400px)" />
-        <source type="image/png" srcset=${background}  media="(min-width: 400px)" />
-        <source type="image/webp" srcset=${background} />
-        <img loading="eager" alt="" type="image/png" src=${background}  width="750" height="375" />
-      </picture>`;
+export const DecorateBlockBackgroundCmp = ({ background = '' }) => html`<img loading="eager" alt="" src=${background} height="1020" width="1920" />`;
```

Comment:
I see the width and height have been significantly increased, might be worth checking if the images used so far scale well with these new values.

Reply:
The sizes chosen were based on the images that have been used in the live quiz. The css sets these at 100% anyway, so a larger width results in less of a potential shift.